README.md

```diff
@@ -52,6 +52,43 @@ FeatureGateMyFeatureName = newFeatureGate("MyFeatureName").
 			mustRegister()
 ```
 
+### defining tests
+Tests are logically associated with FeatureGates.
+When adding any FeatureGated functionality a new test file is required.
+The test files are located in `<group>/<version>/tests/<crd-name>/FeatureGate.yaml`
+```
+route
+  v1
+    tests
+      routes.route.openshift.io
```

Comment:
Punctuation can make it a little more obvious that the ```` ``` ````-block represents a filesystem hierarchy:
````suggestion
The test files are located in `<group>/<version>/tests/<crd-name>/FeatureGate.yaml`:
```
route/
  v1/
    tests/
      routes.route.openshift.io/
````